doc/ledger3.texi

```diff
@@ -6453,7 +6459,7 @@ been executed, such as periodic gathering.
 Redirect the output of ledger to the file defined in @file{FILE}.
 
 @item --pager @var{FILE}
-Specify the pager program to use.
+Direct output to @var{FILE} pager program.
```

Comment:
My first commit was with PROGRAM as you suggested, but I reset it, as a consistency matter. FILE is indeed used consistently in the documentation to tell that this must me a Unix file whatever it is a program, a file, or anything else.